app/views/handlers/individual_response.py

```diff
@@ -132,6 +141,58 @@ def form(self):
     def get_context(self):
         return build_question_context(self.rendered_block, self.form)
 
+    def _get_fulfilment_code(self, mobile_number=None):
```

Comment:
I wonder if this shouldn't be fulfilment_type rather than mobile_number, it makes me think fulfilment codes are usually mobile numbers, perhaps they are. Also its an assumption that if its not mobile it must be postal. Probably more explicit is better here when calling _get_fulfilment_code

Reply:
Addressed in https://github.com/ONSdigital/eq-questionnaire-runner/pull/276#discussion_r487585426